pkg/config/fetcher/from_processes.go

```diff
@@ -23,9 +24,15 @@ func SecurityAgentConfig(config config.Reader) (string, error) {
 		return "", err
 	}
 
+	port := config.GetInt("security_agent.cmd_port")
+	if port <= 0 {
+		return "", fmt.Errorf("invalid security_agent.cmd_port -- %d", port)
+	}
+
 	c := util.GetClient(false)
+	c.Timeout = time.Second * 30
```

Comment:
Should we modify the `GetClient` to allow to get options. Maybe using the option pattern and [variadic arguments](https://gobyexample.com/variadic-functions)  so we can configure the http client `Timeout` directly inside?

Reply:
I had the same question, but configuring `http.Client` is very straightforward so I'm not sure of the gain of this modification.